torchrl/envs/transforms/transforms.py

```diff
@@ -2627,7 +2628,7 @@ class CatFrames(ObservationTransform):
         "dim must be < 0 to accomodate for tensordict of "
         "different batch-sizes (since negative dims are batch invariant)."
     )
-    ACCEPTED_PADDING = {"same", "zeros"}
+    ACCEPTED_PADDING = {"same", "constant"}
```

Comment:
This is BC breaking, can we keep the "zeros" and maybe raise a deprecation warning if it's being used?

Reply:
ok! I just did that